src/pages/[platform]/build-a-backend/data/query-data/index.mdx

```diff
@@ -224,8 +232,21 @@ type Post = Schema['Post']['type'];
 const [posts, setPosts] = useState<Post[]>([]);
 ```
 
+</InlineFilter>
+
+<InlineFilter filters={["angular", "vue"]}>
+
+```ts
+import { type Schema } from '@/amplify/data/resource';
```

Comment:
is this path alias `@` automatically available in angular and vue apps like it is for next.js?